packages/schematics/migrations/update-7-0-0/update-7-0-0.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { Tree, noop } from '@angular-devkit/schematics';
+import * as devkit from '@angular-devkit/schematics';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import { serializeJson } from '../../src/utils/fileutils';
+
+import * as path from 'path';
+
+describe('Update 7.0.0', () => {
```

Comment:
I'd remove this test. It does't give me any confidence that the migration actually works. We should talk about it more in person.